rcl/test/CMakeLists.txt

```diff
@@ -206,6 +206,23 @@ function(test_target_function)
     AMENT_DEPENDENCIES ${rmw_implementation} "osrf_testing_tools_cpp" "test_msgs"
   )
 
+  set(AMENT_GTEST_ARGS "SKIP_TEST")
+  # TODO(wjwwood): remove this when the graph API works properly for connext dynamic
```

Comment:
This TODO doesn't look right, and the condition probably isn't want you want? Also the condition below it looks wrong.